src/components/MoviesList/MoviesList.tsx

```diff
@@ -1,6 +1,13 @@
 import React from 'react';
 import './MoviesList.scss';
 
-export const MoviesList: React.FC = () => (
-  <>Put the list here</>
+import { MovieCard } from '../MovieCard';
+import { Card } from '../Types/Card';
+
+export const MoviesList: React.FC<{ list: Card[] }> = ({ list }) => (
```

Comment:
that's not a common practice to do so, create interface or type
```suggestion
export const MoviesList: React.FC<Props> = ({ list }) => (
```